src/main/kotlin/no/ccat/utils/SearchQueryDataHelper.kt

```diff
@@ -115,6 +116,24 @@ data class QueryParams(val queryString: String = "",
 
 }
 
+
+fun QueryParams.sanitizeQueryStrings() =
+       copy(
+                queryString = queryString.sanitizeForQuery(),
+                lang = lang.sanitizeForQuery(),
+                prefLabel = prefLabel.sanitizeForQuery()
+        )
+
+private fun String.sanitizeForQuery(): String {
+    return trim()
+            .dropWhile {
+                !it.isLetterOrDigit()
+            }
+            .dropLastWhile {
+                (it == '+' || it == ' ')
```

Comment:
Hvis man bruker trim, så gir ikke `it == ' '` mening

Reply:
jo, om en trimma string har whitespace før +